============================= test session starts ==============================
collected 635 items
=================================== FAILURES ===================================
ERROR: invocation failed (exit code 1)
ERROR: invocation failed (exit code 1)
ERROR: invocation failed (exit code 1)
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
ERROR: invocation failed (exit code 1)
E         ================================== log start ===================================
E         =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pyte...on3.9/vendor-packages/tox/__main__.py", line 1, in <module>
ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pyte...on3.9/vendor-packages/tox/__main__.py", line 1, in <module>
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pyte...on3.9/vendor-packages/tox/__main__.py", line 1, in <module>
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
E       ================================== log start ===================================
E       =================================== log end ====================================
=============================== warnings summary ===============================
===Flaky Test Report===
test_parallel_interrupt failed (2 runs remaining out of 3).
test_parallel_interrupt failed (1 runs remaining out of 3).
test_parallel_interrupt failed; it passed 0 out of the required 1 times.
test_parallel_live failed (2 runs remaining out of 3).
ERROR: invocation failed (exit code 1)
ERROR: invocation failed (exit code 1)
ERROR:   a: parallel child exit code 1
ERROR:   b: parallel child exit code 1
test_parallel_live failed (1 runs remaining out of 3).
ERROR: invocation failed (exit code 1)
ERROR: invocation failed (exit code 1)
ERROR:   a: parallel child exit code 1
ERROR:   b: parallel child exit code 1
test_parallel_live failed; it passed 0 out of the required 1 times.
ERROR: invocation failed (exit code 1)
ERROR: invocation failed (exit code 1)
ERROR:   a: parallel child exit code 1
ERROR:   b: parallel child exit code 1
test_parallel_show_output failed (2 runs remaining out of 3).
================================== log start ===================================
=================================== log end ====================================
ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pytest-7/test_parallel_show_output0/pkg123/.tox/e2/log/e2-0.log
================================== log start ===================================
=================================== log end ====================================
ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pytest-7/test_parallel_show_output0/pkg123/.tox/e1/log/e1-0.log
================================== log start ===================================
=================================== log end ====================================
ERROR:   e1: parallel child exit code 1
ERROR:   e2: parallel child exit code 1
ERROR:   e3: parallel child exit code 1
test_parallel_show_output failed (1 runs remaining out of 3).
================================== log start ===================================
=================================== log end ====================================
ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pytest-7/test_parallel_show_output1/pkg123/.tox/e1/log/e1-0.log
================================== log start ===================================
=================================== log end ====================================
ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pytest-7/test_parallel_show_output1/pkg123/.tox/e3/log/e3-0.log
================================== log start ===================================
=================================== log end ====================================
ERROR:   e1: parallel child exit code 1
ERROR:   e2: parallel child exit code 1
ERROR:   e3: parallel child exit code 1
test_parallel_show_output failed; it passed 0 out of the required 1 times.
================================== log start ===================================
=================================== log end ====================================
ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pytest-7/test_parallel_show_output2/pkg123/.tox/e2/log/e2-0.log
================================== log start ===================================
=================================== log end ====================================
ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pytest-7/test_parallel_show_output2/pkg123/.tox/e3/log/e3-0.log
================================== log start ===================================
=================================== log end ====================================
ERROR:   e1: parallel child exit code 1
ERROR:   e2: parallel child exit code 1
ERROR:   e3: parallel child exit code 1
===End Flaky Test Report===
=========================== short test summary info ============================
SKIPPED [1] integration/test_jython_env_create.py needs jython and dev cut of virtualenv
SKIPPED [1] unit/test_z_cmdline.py Platform doesn't support alwayscopy
SKIPPED [1] unit/interpreters/windows/test_pep514.py non posix test
SKIPPED [1] unit/interpreters/windows/test_pep514.py non posix test
SKIPPED [1] unit/interpreters/windows/test_windows.py non posix test
FAILED integration/test_parallel_inception.pytest_parallel_inception - Asse...
FAILED integration/test_parallel_interrupt.pytest_parallel_interrupt - Asse...
FAILED integration/test_provision_int.pytest_provision_interrupt_child[Signals.SIGINT]
FAILED integration/test_provision_int.pytest_provision_interrupt_child[Signals.SIGTERM]
FAILED unit/test_z_cmdline.pytest_tox_console_script - FileNotFoundError: [...
FAILED unit/test_z_cmdline.pytest_tox_quickstart_script - FileNotFoundError...
FAILED unit/interpreters/test_interpreters.pytest_find_alias_on_path - asse...
FAILED unit/session/test_parallel.pytest_parallel - AssertionError: 1
FAILED unit/session/test_parallel.pytest_parallel_live - AssertionError: 1
FAILED unit/session/test_parallel.pytest_parallel_error_report[True] - Asse...
FAILED unit/session/test_parallel.pytest_parallel_error_report[False] - AssertionError: [Kâ ‹ [0] ERROR: invocation failed (exit code 1), logfile: /tmp/pytest-of-andreas/pytest-7/test_parallel_error_report_Fal0/pkg123/.tox/a/log/a-0.log
FAILED unit/session/test_parallel.pytest_parallel_show_output - AssertionEr...
FAILED unit/session/test_parallel.pytest_parallel_no_spinner_on - Assertion...
FAILED unit/session/test_parallel.pytest_parallel_no_spinner_off - Assertio...
FAILED unit/session/test_parallel.pytest_parallel_no_spinner_not_set - Asse...
FAILED unit/session/test_parallel.pytest_parallel_result_json - AssertionEr...
FAILED unit/session/test_parallel.pytest_parallel_result_json_concurrent - ...
FAILED unit/session/test_provision.pytest_provision_cli_args_ignore - Asser...
======= 18 failed, 612 passed, 5 skipped, 1 warning () =======
